stable/kube-ingress-aws-controller/README.md

```diff
@@ -0,0 +1,54 @@
+# kube-ingress-aws-controller
+
+[kube-ingress-aws-controller](https://github.com/zalando-incubator/kube-ingress-aws-controller) is an ingress controller for Kubernetes — the open-source container deployment, scaling, and management system — on AWS.
+
+## TL;DR;
+
+```console
+$ helm repo add incubator http://storage.googleapis.com/kubernetes-charts-incubator
+$ helm install incubator/kube-ingress-aws-controller --namespace kube-system --name aws-ingc
```

Comment:
this is being proposed against stable, not incubator (incubator may be more appropriate - but not my call).

Reply:
Good catch, I'll fix it asap.